SourceLink.Create.BitBucket/SourceLink.Create.BitBucket.csproj

```diff
@@ -17,20 +22,6 @@
       <Pack>true</Pack>
       <PackagePath>build</PackagePath>
     </Content>
-
-    <Content Include="$(OutputPath)netstandard1.4/SourceLink.Create.BitBucket.dll">
-      <Pack>true</Pack>
-      <PackagePath>build/netstandard1.4</PackagePath>
-    </Content>
-    <Content Include="$(OutputPath)netstandard1.4/SourceLink.Create.BitBucket.deps.json">
-      <Pack>true</Pack>
-      <PackagePath>build/netstandard1.4</PackagePath>
-    </Content>
-
-    <Content Include="$(OutputPath)net461/SourceLink.Create.BitBucket.dll">
-      <Pack>true</Pack>
-      <PackagePath>build/net461</PackagePath>
-    </Content>
```

Comment:
Why was this removed? Is it not needed anymore?

Reply:
My understanding is that **\<pack\>** is responsible to generate a nuget package. Why do we need to create a package in the Tests project?

Not to mention I couldn't compile with this in place.

Reply:
This file is `SourceLink.Create.BitBucket.csproj`. If you compare the nupkgs between 2.2.1 and this build, you will see that you are missing those libraries.
https://ci.appveyor.com/project/ctaggart/sourcelink/build/2.2.1/artifacts
https://ci.appveyor.com/project/ctaggart/sourcelink/build/2.2.0-b538/artifacts

Reply:
Ah, ok. I'll have a look at the csproj files changes to ensure only Tests project is affected.